alert.go

```diff
@@ -116,6 +116,8 @@ type AlertNode struct {
 	critsTriggered  *expvar.Int
 
 	bufPool sync.Pool
+
+	level_resets []stateful.Expression
```

Comment:
Go is an opinionated language and prefers camelCase names. Can you change this to `levelResets`?